packages/wallet/src/redux/store.ts

```diff
@@ -1,16 +1,31 @@
 import { applyMiddleware, compose, createStore } from 'redux';
 import { fork } from 'redux-saga/effects';
 import createSagaMiddleware from 'redux-saga';
-
+import * as storage from 'redux-storage';
 const sagaMiddleware = createSagaMiddleware();
 
 import { walletReducer } from './reducer';
 import { sagaManager } from './sagas/saga-manager';
+import filter from 'redux-storage-decorator-filter';
+import createEngine from 'redux-storage-engine-localstorage';
 
+const storageEngine = filter(createEngine('magmo-wallet'), [
+  'whitelisted-key',
+  ['address'],
+  ['privateKey'],
+  ['channelStore'],
+]);
+const storageMiddleware = storage.createMiddleware(storageEngine, [
```

Comment:
LocalStorage gets updated every time an action is dispatched. We exclude the `CHANNEL_UPDATE` action since that action should only be dispatched in response to loading from storage. Ideally if we're only storing `channelStore` we should only whitelist actions that update `channelStore`